### ISSUE TITLE
Возможность отключить запись в "Журнал регистрации" событий по определенным объектам метаданных

### ISSUE THREAD

**Issue by ptizaezhik** [SeiOkami/OneS#79]
На текущий момент в "Журнале регистрации" фиксируются изменения всех объектов метаданных.
При этом, в каждой конфигурации существует огромное число служебных регистров, справочников, документов, история по которым не нужна, а записей по ним - колоссальное количество.
Как пример - любой регистр сведений, используемый в качестве лога.
Если будет возможность отключать по определенным объектам метаданных запись в журнал регистрации, это сильно упростит работу с журналом.

**Comment by SeiOkami** [comment]
Возможно я ошибаюсь, но это реализовано в платформе.
Посмотрите описание метода УстановитьИспользованиеСобытияЖурналаРегистрации()

**Comment by tormozit** [comment]
Да. Уже сто лет можно настраивать через ИР например http://devtool1c.ucoz.ru/index/nastrojka_zhurnala_registracii/0-13

**Comment by tormozit** [comment]
Касательно записи настроить выборочное логирование методом УстановитьИспользованиеСобытияЖурналаРегистрации() нельзя. Можно только для событий "Доступ" и "Отказ доступа". Можно например в ИР посмотреть какие есть возможности у этого метода http://devtool1c.ucoz.ru/index/nastrojka_zhurnala_registracii/0-13

**Comment by SeiOkami** [comment]
Подсказали ссылку:

https://its.1c.ru/db/v8319doc#bookmark:dev:TI000000826

**Comment by SeiOkami** [comment]
Опубликовано на канале: https://t.me/UnionOneS/61
Отправлено в чат: https://t.me/e1c_community/48305
Отправлено боту 1С в 07:33
